### PR TITLE
[ru] Update garbage collection page

### PR DESCRIPTION
**Pull request by Zhurik**

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
Please sign CLA
https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by Zhurik** [comment]
> Please sign CLA https://github.com/kubernetes/community/blob/master/CLA.md

done

**Review by Arhell — COMMENTED** [review]
/lgtm
/assign @Potapy4

File: content/ru/docs/concepts/architecture/garbage-collection.md [hunk 1]
```diff
@@ -131,4 +131,4 @@ Kubelet собирает мусор только у контейнеров, ко
 
 * Узнайте больше о [ownership of Kubernetes objects](/docs/concepts/overview/working-with-objects/owners-dependents/).
 * Узнайте больше о Kubernetes [finalizers](/docs/concepts/overview/working-with-objects/finalizers/).
-* Узнать о [TTL контроллере](/docs/concepts/workloads/controllers/ttlafterfinished/) (beta) that cleans up finished Jobs.
\ No newline at end of file
+* Узнать о [TTL контроллере](/docs/concepts/workloads/controllers/ttlafterfinished/) (beta) that cleans up finished Jobs.
```
Comment (sftim) [inline content/ru/docs/concepts/architecture/garbage-collection.md:134]: I recommend translating “that cleans up finished Jobs.” into Russian.

**Comment by shurup** [comment]
This whole document requires an overall text correction since it has lots of typos, problems with commas (some are unnecessary, others are missing), etc. @Zhurik, would you be interested in editing it in general?

**Comment by Zhurik** [comment]
> This whole document requires an overall text correction since it has lots of typos, problems with commas (some are unnecessary, others are missing), etc. @Zhurik, would you be interested in editing it in general?

I'm okay with it. Will edit it later

**Comment by shurup** [comment]
Hi @Zhurik! Will you be able to proceed with correcting this document?

**Comment by divya-mohan0209** [comment]
Hello @Zhurik , thank you for submitting changes via the PR. However, due to no activity in ~ a month and no response, we will be closing the PR. Please consider reopening it whenever you have cycles to work on the change.
/close

**Pull request closed without merging**